testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts

```diff
@@ -13,7 +13,8 @@ export enum CredentialTypes {
 
 export enum ProgramLengthOptions {
   WeeksToLessThanYear = "12WeeksToLessThan1Year",
-  OneToTwoYears = "1YearToLessThan2Years",
+  WeeksTo59Weeks = "52WeeksTo59Weeks",
```

Comment:
I know we cant name the enum variables with a number , but now as we have 3 variables starting from weeks to lessthanyear, 59weeks or 2 years. I think we can create the variable more descriptively like
 Weeks_52_TO_59 = "52WeeksTo59Weeks"
 Weeks_12_TO_1_Year = "12WeeksToLessThan1Year"
 Weeks_60_TO_2_Years = "60WeeksToLessThan2Years"

Reply:
@guru-aot Do we use the cypress files anymore. If not, then I will revert my change.

Reply:
Ho i am sorry i got confused with the existing e2e. All good